content/_data/events/2020-01-30-fosdem.adoc

```diff
@@ -0,0 +1,13 @@
+---
+
+title: "Jenkins at FOSDEM"
+location: "Brussels, Belgium"
+date: "2020-01-30T8:30:00"
+endDate: "2020-02-02T17:00:00"
+link: "/events/fosdem"
+
+---
+
+Meet us at FOSDEM!
+We will have a stand there on Feb 01-02, and we are also going to have a contributor summit on Jan 31st.
```

Comment:
```suggestion
There will be a Jenkins stand on Feb 01-02, CI/CD devroom on Feb 02, contributor summit on Jan 31st and Jenkins / Jenkins X trainings on Jan 30.
```

WDYT @olblak ?

Reply:
What do you think to put them in chronological order and with links to the various pages

`There will be link:https://www.eventbrite.com/e/jenkins-pipeline-fundamentals-training-tickets-87080214265[Jenkins]/link:https://www.eventbrite.com/e/workshop-cloud-native-kubernetes-first-serverless-continuous-delivery-with-jenkins-x-kubernetes-and-tickets-87082627483[JenkinsX] trainings on Jan 30, a Jenkins contributor link:https://www.meetup.com/jenkinsmeetup/events/267684785/[summit] on Jan 31, and during link:https://fosdem.org/[Fosdem] on Feb 01-02, there will be a Jenkins stand and a CI/CD devroom on link:https://fosdem.org/2020/schedule/track/continuous_integration_and_continuous_deployment/[Sunday]. `

Reply:
Links do not work in this layout. I put them in an order of relevance